sanic/src/train_RNN.py

```diff
@@ -14,32 +14,35 @@
 NUM_FEATURES = 2048
 EPOCHS = 50
 
+# Define CLI arguments
 ap = argparse.ArgumentParser()
 ap.add_argument("-l", "--load", required=True,
-	help="load pickle data, or create pickle data")
-ap.add_argument("-p", "--picture", required=True,
-	help="What to call picture created from training history")
-ap.add_argument("-m", "--model", required=True,
-	help="What to call model file")
+	help="load pickle data (True), or create pickle data (False)")
+ap.add_argument("-n", "--name", required=True,
+	help="What to call picture and model created from training")
 args = vars(ap.parse_args())
 
-
+# Get training and test data ids and labels
 train_data, test_data = utils.get_data_frame_dicts()
 
+# Setup the training and test ids and labels as a DataFrame (Match the id to the label)
 train_df = pd.DataFrame(train_data)
 test_df = pd.DataFrame(test_data)
 
+print(f"[INFO] Total number of videos for training: {len(train_df)}")
+print(f"[INFO] Total number of videos for testing: {len(test_df)}")
+
+# Map the unique labels of the training data to integer indices
 label_processor = keras.layers.StringLookup(
     num_oov_indices=0, vocabulary=np.unique(train_df["label"])
 )
 
-
 print("[INFO] loading feature extractor...")
 feature_extractor = utils.build_feature_extractor()
 
-
 if args["load"] == "True":
-
+    # Load the training and test data and labels from the stored pkl files
+    print("[INFO] loading pickle data...")
     with open("./data/traindata.pkl", 'rb') as file:
         train_data = pickle.load(file)
```

Comment:
Bare en mere general tanke:
Skal vi samle de forskellige modeller i en fil? Eller er det ikke bare nemest at holde dem hver for sig?
Kom til at tænke på det i forhold til at hardcode fil navne på pickle

Reply:
Man burde vel nok bare kunne give en fil/path som parameter istedet for at hardcode filepaths?

Reply:
Er det noget der skal laves nu? Eller skal vi vente med det?

Reply:
Tror du det er svært?